ftx/medical/clinical_ontology.py

```diff
@@ -486,8 +486,8 @@ class Hyponym(Link):
 
     hyponym_link: Optional[str]
 
-    ParentType = Token
-    ChildType = Token
+    ParentType = Phrase
+    ChildType = Phrase
```

Comment:
Is this a manual change of ontology class? We should avoid this and change through json config.

Reply:
No this is restored to the current version checked into master to be able to compile correctly so the change to Phrase is correct and will merge with no problem (now after changed a comment line 2 it is identical with master)